From b0a78dba46040652a8d1ab54f6787d18831ffefd Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 11 Aug 2014 12:26:32 -0400
Subject: [mfd] lpc_ich: Convert ICH GPIOs IDs to enum

Message-id: <1407759999-16143-2-git-send-email-prarit@redhat.com>
Patchwork-id: 87595
O-Subject: [RHEL7.1 PATCH BZ 1084134 1/8] mfd: lpc_ich: Convert ICH GPIOs IDs to enum
Bugzilla: 1084134
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1084134

commit 6cec365e3eba3dd8c864056d8d3fd9e73ab8dd7a
Author: Vincent Donnefort <vdonnefort@gmail.com>
Date:   Fri Feb 14 15:01:53 2014 +0100

    mfd: lpc_ich: Convert ICH GPIOs IDs to enum

    All those IDs are arbitrary and so can be encapsulated into an enumeration.

    Signed-off-by: Vincent Donnefort <vdonnefort@gmail.com>
    Signed-off-by: Lee Jones <lee.jones@linaro.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/mfd/lpc_ich.h b/include/linux/mfd/lpc_ich.h
index 3e1df64..293b062 100644
--- a/include/linux/mfd/lpc_ich.h
+++ b/include/linux/mfd/lpc_ich.h
@@ -31,13 +31,15 @@
 #define ICH_RES_GPE0 1
 
 /* GPIO compatibility */
-#define ICH_I3100_GPIO  0x401
-#define ICH_V5_GPIO  0x501
-#define ICH_V6_GPIO  0x601
-#define ICH_V7_GPIO  0x701
-#define ICH_V9_GPIO  0x801
-#define ICH_V10CORP_GPIO 0xa01
-#define ICH_V10CONS_GPIO 0xa11
+enum {
+ ICH_I3100_GPIO,
+ ICH_V5_GPIO,
+ ICH_V6_GPIO,
+ ICH_V7_GPIO,
+ ICH_V9_GPIO,
+ ICH_V10CORP_GPIO,
+ ICH_V10CONS_GPIO,
+};
 
 struct lpc_ich_info {
  char name[32];
-- 
1.7.1